server/src/domain/album/album-response.dto.ts

```diff
@@ -23,6 +23,8 @@ export class AlbumResponseDto {
   startDate?: Date;
   endDate?: Date;
   isActivityEnabled!: boolean;
+  @ApiProperty({ enumName: 'AssetOrder', enum: AssetOrder })
+  order!: AssetOrder;
```

Comment:
I am pretty sure it will break the mobile app because the SDK will check for this property and it will be null from previous version, so the users will not see any albums

Reply:
Let's make it optional then to avoid the breaking change. We can change it back in the future after a few releases.

Reply:
> @shenlong-tanwen will this new property break the mobile app? Do we need to make it `order?: AssetOrder`?

Yes. The mobile app will not sync albums with older server versions. Making it optional would be best.